.vscodeignore

```diff
@@ -0,0 +1,2 @@
+# TS build files
+build
```

Comment:
The place where this is more commonly set is in the [`.vscode/settings.json`](https://github.com/artsy/kaws/blob/master/.vscode/settings.json#L6-L12) file.

Reply:
Yup, just so that all of our config is consolidated in one place. Here's how we have it [set in reaction](https://github.com/artsy/reaction/blob/master/.vscode/settings.json#L2-L11).

Reply:
Which brings up the thought -- in Reaction and elsewhere we use `dist` as the folder, where here we use `build` -- it's not a big deal but the more alignment we have the better. Thoughts @l2succes? (Either way, not something for this pr)

Reply:
Yeah good point @damassi, I don’t even think this file worked for us when we added it.